runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixOverlay.java

```diff
@@ -54,20 +55,38 @@ public Dimension render(Graphics2D graphics)
 	{
 		plugin.getStreams().forEach((object, timer) ->
 		{
-			final ProgressPieComponent progressPie = new ProgressPieComponent();
-			progressPie.setDiameter(15);
-			progressPie.setFill(CYAN_ALPHA);
-			progressPie.setBorderColor(Color.CYAN);
-			progressPie.setPosition(object.getCanvasLocation(100));
+			if (object.getId() == 6661 || object.getId() == 6665)
+			{
+				final ProgressPieComponent progressPie = new ProgressPieComponent();

```

Comment:
Don't duplicate this code for the different object IDs. Only switch the colors for the different IDs.